client/blocks/login/style.scss

```diff
@@ -9,6 +9,57 @@
 	}
 }
 
+.layout.is-jetpack-woocommerce-flow {
+	.login__jetpack-logo {
+		text-align: center;
+		border-bottom: 1px solid $muriel-gray-50;
```

Comment:
Hey! Out of curiosity, is there a reason why you’re using a sass var in this case instead of css custom props?

Reply:
No reason -- thank you for catching that. I've fixed it up.

Reply:
Thanks! :+1: